test/CMakeLists.txt

```diff
@@ -136,13 +136,29 @@ if( iodaconv_bufr_ENABLED )
     testinput/bufr_tables
     testinput/gdas.t18z.1bmhs.tm00.bufr_d
     testinput/gdas.t00z.1bhrs4.tm00.bufr_d
+    testinput/gdas.t00z.airsev.tm00.bufr_d
+    testinput/gdas.t00z.1bamua.tm00.bufr_d
+    testinput/gdas.t00z.avcsam.tm00.bufr_d
+    testinput/gdas.t00z.atms.tm00.bufr_d
```

Comment:
Bufr files are very large... We like to keep them around 200 k or so. You can try the subset_bufr.x utility that Rahul wrote a while ago in order to shrink these files does https://github.com/aerorahul/bufr-tools 

usage: >> subset_bufr.x **<input_file>** **<output_file>** **<percent_of_file_to_keep>**
example: >> `subset_bufr.x input.bufr_d output.bufr_d 10`    # Keeps 10 percent of the messages in the file.

Reply:
If subset_bufr.x gives you problems (its problematic for some files), Emily has a different way of doing it which might work better.